manifests/module/ldap.pp

```diff
@@ -63,18 +126,20 @@
   Integer $idle_timeout                                               = 60,
   Optional[Float] $connect_timeout                                    = undef,
 ) {
-  $fr_package          = $::freeradius::params::fr_package
-  $fr_service          = $::freeradius::params::fr_service
-  $fr_modulepath       = $::freeradius::params::fr_modulepath
-  $fr_basepath         = $::freeradius::params::fr_basepath
-  $fr_group            = $::freeradius::params::fr_group
+  require freeradius::params
```

Comment:
Normally this should be
```suggestion
  require freeradius
```
but when doing it the spec tests are failing because of `Duplicate declaration: File[/etc/raddb]`. I did not found the issue for that one. That is why i keep it here with `freeradius::params` even so the variable `$freeradius::fr_3_1` is used here.

Reply:
I've split these issues out into #200 and #201 to be addressed separately